easybuild/easyconfigs/j/jax/jax-0.2.19-foss-2020b.eb

```diff
@@ -56,7 +47,7 @@ components = [
         'use_pip': True,
     }),
     ('jaxlib', local_jaxlib_ver, {
-        'easyblock': 'PythonPackage',
+        'easyblock': 'EB_jaxlib',
```

Comment:
When https://github.com/easybuilders/easybuild-easyblocks/pull/2543 is merged, this can be removed too.